src/commcare_cloud/fab/README.md

```diff
@@ -47,13 +47,13 @@ www/<env>/current -> www/<env>/releases/2015-09-10_04.00
 To initiate a regular deploy, use the following command:
 
 ```
-fab <env> deploy
+cchq <env> [--control] deploy
```

Comment:
does control option deploy from control machine??

Reply:
yep

Reply:
nice! I didn't know about that.